backend/app/treasury/actions.py

```diff
@@ -139,8 +156,10 @@ def update_treasury_assets_risk_contributions(
 
 
 async def build_treasury_with_assets(
-    treasury_address: str, chain_id: int, start: str, end: str
+    request_params: tuple[tuple[str, int], str, str]
```

Comment:
Sure, initially I wanted to add more params to `build_treasury_with_assets` and I thought packing the params into one would make for a cleaner function signature.

I have since removed these extra params but I still maintained that using a single param is cleaner for this function.